pc2cps

```diff
@@ -72,18 +86,60 @@ class pkgconfig(object):
             else:
                 deps[dep[0]] = {}
 
-        return deps
+        self.requires = deps
 
     #--------------------------------------------------------------------------
     @property
     def component(self):
         def _keys(x):
-            return None if x is None else x.keys()
-
-        c = {
-            'Type': 'interface',
-        }
-        extend(c, 'Link-Flags', self._get_split('Libs'))
+            return [] if x is None else x.keys()
+
+        def _list(re_):
+            ma = [re_.match(x) for x in link_flags]
```

Comment:
Please use `mf` here instead of `ma`; → Match Flag. (Or just use `m` if there is no need to differentiate between different matches.) Similar for other instances of `ma`.